packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -1187,6 +1187,18 @@ export default class Store extends EventEmitter<{|
       console.groupEnd();
     }
 
+    const indicesOfCachedErrorsOrWarningsAreStale =
+      !haveErrorsOrWarningsChanged &&
+      (addedElementIDs.length > 0 || removedElementIDs.size > 0);
+    if (indicesOfCachedErrorsOrWarningsAreStale) {
+      this._cachedErrorAndWarningTuples.forEach(entry => {
+        const index = this.getIndexOfElementID(entry.id);
+        if (index !== null) {
```

Comment:
Do we need to guard here or would this be bug at this point and we should include a warning?

Reply:
I don't think this should ever be able to happen, and we'd just have this check to satisfy Flow.